frontend/src/Sidebar.res

```diff
@@ -11,30 +11,51 @@ let pullToString = ((pull_number, branch)) =>
   | None => "#" ++ Belt.Int.toString(pull_number)
   }
 
-let pullsMenu = (~pulls) => {
-  <Column>
-    <Text color=Sx.gray700 weight=#bold uppercase=true size=#md> {Rx.text("Pull Requests")} </Text>
-    {pulls
-    ->Belt.Array.mapWithIndex((i, pull) =>
-      <Link key={string_of_int(i)} href={linkForPull(pull)} text={pullToString(pull)} />
-    )
-    ->Rx.array}
-  </Column>
+module PullsMenu = {
+  @react.component
+  let make = (~pulls, ~repo, ~selectedPull=?) => {
+    <Column>
+      <Text color=Sx.gray700 weight=#bold uppercase=true size=#md>
+        {Rx.text("Pull Requests")}
+      </Text>
+      {pulls
+      ->Belt.Array.mapWithIndex((i, pull) => {
+        let (pull_number, _) = pull
+        <Link
+          active={selectedPull->Belt.Option.mapWithDefault(false, selectedPullNumber =>
+            selectedPullNumber == pull_number
+          )}
+          key={string_of_int(i)}
+          href={linkForPull(repo, pull)}
+          text={pullToString(pull)}
+        />
+      })
+      ->Rx.array}
+    </Column>
+  }
 }
 
 @react.component
-let make = (~url: ReasonReact.Router.url, ~pulls, ~onSynchronizeToggle, ~synchronize) => {
-  let repo_name = "mirage/index"
+let make = (
+  ~pulls,
+  ~selectedRepo,
+  ~repos,
```

Comment:
The backend and the db use `repo_id` to represent `$org_name/$repo_name`. Can we keep this convention here and call this prop `repo_ids` instead of `repos`?